test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -633,3 +634,13 @@ func getCustomVolumeFactory(pvTemplate *v1.PersistentVolume) func(id int) *v1.Pe
 		return pv
 	}
 }
+
+func isRetryableAPIError(err error) bool {
+	// These errors may indicate a transient error that we can retry in tests.
+	if apierrors.IsInternalError(err) || apierrors.IsTimeout(err) || apierrors.IsServerTimeout(err) {
+		return true
+	}
+	// If the error sends the Retry-After header, we respect it as an explicit confirmation we should retry.
+	_, shouldRetry := apierrors.SuggestsClientDelay(err)
```

Comment:
I would be ok with a change to client-go to add an option to retry on 429s if the retry is within the timeout of the request.

Reply:
https://github.com/kubernetes/kubernetes/blob/e64ebe01314b41fa68cff827a31ab133ec6963a4/staging/src/k8s.io/client-go/rest/request.go#L870

@deads2k i believe that's the current behavior of client-go, unless you opt-out using https://github.com/kubernetes/kubernetes/pull/89566. we don't need to manually check the Retry-After header here.

Reply:
@yue9944882 does that mean this check is not necessary on client side?

Reply:
@Huang-Wei yea that's the point

Reply:
Thanks @yue9944882. Dropped this check.